### PR TITLE
BLUI-2970 Design Release

### PR DESCRIPTION
**Pull request by huayunh**
## Changes proposed in this Pull Request:
- Add release notes from the design side. 
- Leaving the dev notes to you guys
- Add João to the contributor list

## Screenshots:
image

## To Test:
- `yarn start`
- view the result on the landing page and `http://localhost:3000/release-notes`

### CODE REVIEW

**Comment by ektasawant** [comment]
Changes looks good 👍 .

File: src/docs/release-notes/R23/FullNotes.mdx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+-   Added more card components to the [Component Sticker Sheet](https://www.figma.com/community/file/1024360297793425107) on Figma.
+-   Added more customization options for the Figma plugin [Variant Switcher](https://www.figma.com/community/plugin/971482182464094790).
+-   Added a [video tutorial](https://www.youtube.com/watch?v=JBiuas4_Occ) for Variant Switcher.
+
+### Miscellaneous
+
+-   Bug fixes & minor updates
+
+### Package Summary
```
Comment (emclaug2) [inline src/docs/release-notes/R23/FullNotes.mdx:33]: Is this Package Summary supposed to be here? There's currently no notes below it.

Comment (huayunh) [reply src/docs/release-notes/R23/FullNotes.mdx:33]: This PR is just for design notes; leaving this section here to remind devs to fill it in.

**Review by joebochill — APPROVED** [review]
Looks good for a starting point

**Pull request merged**